src/style.css

```diff
@@ -1 +1,118 @@
-/* Styles go here */
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(fonts/Roboto-Medium.ttf);
+}
+
+.html {
+  box-sizing: border-box;
+
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  margin: 0;
+}
+
+.link-reset {
+  color: inherit;
+  text-decoration: none;
+}
+
+.lires {
+  margin: 0;
+  padding: 0;
+
+  list-style-type: none;
+  line-height: 1;
+}
+
+.container {
+  max-width: 1200px;
+
+  margin: 0 auto;
+  padding: 0 50px;
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo {
+  display: block;
+
+  object-fit: contain;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+
+  height: 100%;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  position: relative;
+
+  display: block;
+  align-items: center;
+  justify-content: center;
+
+  height: 100%;
+  padding: 24px 0;
+
+  color: #000;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-align: center;
+  text-decoration: none;
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+
+  transition: color 0.3s;
```

Comment:
Transition is cool, but there is no need to set it here, since you've set it to `nav__link` class